.teamcity/Gradle_Check/configurations/BuildJavaVersions.kt

```diff
@@ -0,0 +1,7 @@
+package configurations
+
+
+val buildJavaHome = "%linux.java9.oracle.64bit%"
+val coordinatorPerformanceTestJavaHome = "%linux.java8.oracle.64bit%"
+val individualPerformanceTestJavaHome = "%linux.java8.oracle.64bit%"
```

Comment:
A comment: The performance tests were using a different Java version, `/opt...-latest`. I am fine with changing this to the default java8 one we use everywhere else.

Reply:
I've confirmed they're pointed to same java directory. `/opt/..-latest` is equivalent to `%linux.java8.oracle.64bit%`.